python/lsst/pipe/tasks/dcrAssembleCoadd.py

```diff
@@ -392,6 +399,12 @@ def run(self, skyInfo, tempExpRefList, imageScalerList, weightList,
                                                                   spanSetMaskList,
                                                                   stats.ctrl)
                     convergenceCheck = (convergenceList[-1] - convergenceMetric)/convergenceMetric
+                    if convergenceCheck < 0:
+                        self.log.warn("Coadd %s diverged before reaching maximum iterations or"
+                                      " desired convergence improvement of %s."
+                                      " Divergence: %s",
+                                      subBBox, self.config.convergenceThreshold, convergenceCheck)
```

Comment:
Is this a "warning", or "error" state? If it diverges, should the code raise an exception, or can things further on continue?

Reply:
I believe it should be just a warning, because the model will typically have converged well overall and should be fine to use for making templates for image differencing.